cloud/organization/organization.go

```diff
@@ -0,0 +1,174 @@
+package organization
+
+import (
+	http_context "context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"strconv"
+
+	"github.com/pkg/errors"
+
+	astro "github.com/astronomer/astro-cli/astro-client"
+	"github.com/astronomer/astro-cli/cloud/auth"
+	"github.com/astronomer/astro-cli/config"
+	"github.com/astronomer/astro-cli/pkg/httputil"
+	"github.com/astronomer/astro-cli/pkg/input"
+	"github.com/astronomer/astro-cli/pkg/printutil"
+)
+
+var (
+	httpClient = httputil.NewHTTPClient()
+
+	errInvalidOrganizationKey  = errors.New("invalid organization selection")
+	errInvalidOrganizationName = errors.New("invalid organization name")
+	AuthLogin                  = auth.Login
+)
+
+type OrgRes struct {
+	ID            string   `json:"id"`
+	Name          string   `json:"name"`
+	AuthServiceID string   `json:"authServiceId"`
+	Domains       []string `json:"domains"`
+	ShortName     string   `json:"shortName"`
+	CreatedAt     string   `json:"createdAt"`
+}
+
+func newTableOut() *printutil.Table {
+	return &printutil.Table{
+		Padding:        []int{44, 50},
+		DynamicPadding: true,
+		Header:         []string{"NAME", "ID"},
+		ColorRowCode:   [2]string{"\033[1;32m", "\033[0m"},
+	}
+}
+
+var ListOrganizations = func(c config.Context) ([]OrgRes, error) {
```

Comment:
nitpick: not sure we need to make this function a variable, since the only logic inside is to make an API call, which can be mocked easily by overriding `httpClient` itself. Ex: https://github.com/astronomer/astro-cli/blob/main/cloud/auth/auth_test.go#L101-L107

Nothing wrong with making a function variable, but we should avoid it as much as possible so that folks in the future might not end up misusing it.

Reply:
okay I updated the tests